jni/nnstreamer.mk

```diff
@@ -60,6 +60,10 @@ NNSTREAMER_DECODER_DV_SRCS := \
 NNSTREAMER_DECODER_IL_SRCS := \
     $(NNSTREAMER_EXT_HOME)/tensor_decoder/tensordec-imagelabel.c
 
+# decoder pose estimation
```

Comment:
* Discussion Question:
In the past, we added the "./jni" folder as a basic set to provide how to make JNI interface with nnstreamer for Android platform. do we have to continually keep nnstreamer/jni/ and nnstreamer-example/android/..../jni/?

Reply:
I think we have to if we want nnstreamer to be the shared lib.